pom.xml

```diff
@@ -60,7 +60,7 @@
       <plugin>
         <groupId>com.google.cloud.tools</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
-        <version>0.1.0-beta</version>
+        <version>1.2.2-SNAPSHOT</version>
```

Comment:
we don't want snapshot versions in this repo. Lets just update this post-release